lib/react_client.dart

```diff
@@ -493,20 +511,33 @@ final ReactDartInteropStatics2 _dartInteropStatics2 = (() {
         component.componentWillReceiveProps(new JsBackedMap.backedBy(jsNextProps));
       });
 
-  void _updatePropsAndStateWithJs(Component2 component, JsMap props, JsMap state) {
+  void _updatePropsAndStateWithJs(Component2 component, JsMap props, JsMap state, [dynamic context]) {
     component
       ..props = new JsBackedMap.backedBy(props)
       ..state = new JsBackedMap.backedBy(state);
+    if (context != null) {
```

Comment:
This shouldn't be wrapped in a conditional, to accommodate context changing from a non-null value to `null`.